### ISSUE TITLE
Run the code to generate a summary for a text

### ISSUE THREAD

**Issue by rudra0713** [jiacheng-xu/DiscoBERT#8]
Hi, 
   Great work in article summarization. Can you kindly tell me, if I want to generate the summary for a single random article, which file should I run?

**Comment by chetanambi** [comment]
@zhegan27, @jiacheng-xu 

I have been looking into summarization using coreference resolution for sometime and stumbled upon this. Really awesome work work on this. I wanted to try out DiscoBERT but didn't find details on how to do it. Could you please provide steps for generating the summary.

**Comment by jiacheng-xu** [comment]
Hi all, https://utexas.app.box.com/v/DiscoBERT-ACL2020/folder/110710107654 is where I put the model configuration file. have you guys checked this folder?

**Comment by chetanambi** [comment]
Hi @jiacheng-xu, Thanks for getting back. I did look into the utexas link and didn't information on how to use the code to generate the summary. I would appreciate you could provide steps for generating the summary.

**Comment by rudra0713** [comment]
I agree with @chetanambi. For example, given an article, a script that takes the article as input, applies a pretrained model and generates the summary, or even a guide to train the model from scratch and then test on a datset would be super beneficial.

**Comment by chetanambi** [comment]
@jiacheng-xu Could you please provide the steps for generating the summary. Lot of people are interested in trying DiscoBERT. Really appreciate your response on this.

**Comment by rudra0713** [comment]
Hi, is there any update on this?

**Comment by YTZ01** [comment]
> Hi, is there any update on this?

@rudra0713 @chetanambi Hello, I wonder if u know how to train the model from scratch, I want to learn this project but I don't know the command to start.
